.github/workflows/github-action-checks.yml

```diff
@@ -28,4 +28,4 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Check spelling
-        uses: crate-ci/typos@master
+        uses: crate-ci/typos@v1.31.0
```

Comment:
v1.31.2 is the current release of typos.

I think it's fine to leave it at master rather than periodically upgrading the versions. 

This repository isn't a codebase that needs to be compiled, so having reproducible builds isn't a rationale here.